bedrock/firefox/templates/firefox/australis/fx38_0_5/firstrun.html

```diff
@@ -23,10 +23,7 @@
 
 {% block body_class %}mozID{% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
-{% block site_css %}
+{% block page_css %}
   {% stylesheet 'firefox_fx38_0_5_firstrun' %}
```

Comment:
This bundle needs pruning - has its own reference to `sandstone.less` and `tabzilla-static.less`.